make sure easybuild flags and libs are used for llvmlite
March 7th 2018 by Sam Moors (Vrije Universiteit Brussel - VUB)
April 2nd 2019 updated for 0.28 - Josef Dvoracek (Institute of Physics - Czech Academy of Sciences)
--- /home/users/jose/.local/easybuild/build/numba/0.43.1/foss-2018b-Python-2.7.15/llvmlite/llvmlite-0.28.0/ffi/Makefile.linux.orig      2019-04-02 14:20:02.228129000 +0200
+++ /home/users/jose/.local/easybuild/build/numba/0.43.1/foss-2018b-Python-2.7.15/llvmlite/llvmlite-0.28.0/ffi/Makefile.linux   2019-04-02 14:21:12.629231000 +0200
@@ -5,9 +5,9 @@
 CXX_FLTO_FLAGS ?= -flto
 LD_FLTO_FLAGS ?= -flto -Wl,--exclude-libs=ALL
 
-CXXFLAGS = $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
+CXXFLAGS := $(CXXFLAGS) $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
 LDFLAGS := $(LDFLAGS) $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
-LIBS = $(LLVM_LIBS)
+LIBS := $(LIBS) $(LLVM_LIBS)
 INCLUDE = core.h
 SRC = assembly.cpp bitcode.cpp core.cpp initfini.cpp module.cpp value.cpp \
          executionengine.cpp transforms.cpp passmanagers.cpp targets.cpp dylib.cpp \
